<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 Appgulim</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>계산하기</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="displayResult">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 Appgulim</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>등급</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadSetting">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 Appgulim</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>세팅 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingSubject">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 Appgulim</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>과목 세팅</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="displaySubject">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>324</width>
     <height>471</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <row/>
   <column>
    <property name="text">
     <string>과목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>등급</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
